### PR TITLE
Switched to Yiisoft\Yii\Bootstrap4 namespace and small documentation changes

### PR DESCRIPTION
**Pull request by schmasterz**
| Q             | A
| ------------- | ---
| Breaks BC?    | yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,14 +63,13 @@
...
         "psr-4": {
-            "yii\\bootstrap4\\": "src"
+            "Yiisoft\\Yii\\Bootstrap4\\": "src"
         }
     },
     "autoload-dev": {
         "psr-4": {
             "yii\\captcha\\tests\\": "vendor/yiisoft/yii-captcha/tests",
-            "yii\\bootstrap4\\tests\\": "tests"
-        }
```
Comment (machour) [inline composer.json]: woopsy

**Comment by schmasterz** [comment]
Yes, I've just saw Travis error I will fix it in few minutes. What about yii-gii. I think that it also requires Bootstrap namespace changing?

**Comment by machour** [comment]
yii-gii have been rewritten to be bootstrap free: https://github.com/yiisoft/yii2-gii/blob/master/CHANGELOG.md#210-march-17-2019

File: tests/TestCase.php [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+namespace Yiisoft\Yii\Bootstrap4\Tests;
 
 use yii\di\Container;
 use Yiisoft\Arrays\ArrayHelper;
 
 /**
  * This is the base class for all yii framework unit tests.
  */
-abstract class TestCase extends \yii\tests\TestCase
+abstract class TestCase extends \yii\Tests\TestCase
```
Comment (machour) [inline tests/TestCase.php]: should be \yii\tests\TestCase

**Comment by machour** [comment]
Last problem got to do with: https://github.com/yiisoft/yii-bootstrap4/blob/master/config/tests.php#L7-L9

Merging this PR and having a look locally, thank you again!

**Pull request merged**